src/api/GitHub.js

```diff
@@ -373,7 +373,7 @@ export default class GitHub {
             throw e;
         }
 
-        return results.sort((a, b) => a.updated_at < b.updated_at);
+        return results.sort((a, b) => b.updated_at - a.updated_at);
```

Comment:
From the network you have strings: `updated_at: "2018-10-03T12:13:12Z"`. If you don't map them to date objects, neither comparison nor subtraction is correct. You should parse dates and compare them.

Reply:
![image](https://user-images.githubusercontent.com/106999/54900203-2dddad00-4ef4-11e9-807b-5e43ea0f9c67.png)
subtraction however is less correct 😅

Reply:
Actually I fixed sorting :) All `update_at` here are date objects.